demo/projects/lit-element/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "@web/demo-lit-element",
+  "version": "0.0.0",
+  "description": "Webcomponent lit-element following open-wc recommendations",
+  "license": "MIT",
+  "author": "my-element",
+  "main": "index.js",
+  "module": "index.js",
+  "scripts": {
+    "start": "es-dev-server --app-index demo/index.html --node-resolve --open --watch",
+    "test": "wtr test/**/*.test.js --coverage",
```

Comment:
I would prefer to using web-test-runner in our demos and documentation as acronyms are always tough especially for new users

Reply:
done